src/Misc/Part.cpp

```diff
@@ -597,7 +597,7 @@ void Part::NoteOff(note_t note) //release the key
 }
 
 void Part::PolyphonicAftertouch(note_t note,
-				unsigned char velocity)
+                unsigned char velocity)
```

Comment:
Looks like an unintentional whitespace change (also same for the other change in Part.cpp within this commit).